dockers/docker-sonic-telemetry/supervisord.conf

```diff
@@ -14,7 +20,7 @@ stderr_logfile=syslog
 [program:rsyslogd]
 command=/usr/sbin/rsyslogd -n
 priority=2
-autostart=false
+autostart=true
```

Comment:
You modified the wrong line here. You should have changed `autorestart=true`. Leave `autostart=false` as rsyslogd is started in the start.sh script.

Reply:
Done!